pegasus/forms/hoc_hardware_prizes_2015.rb

```diff
@@ -0,0 +1,26 @@
+require pegasus_dir 'forms/hoc_hardware_prizes_2014'
+
+class HocHardwarePrizes2015 < HocHardwarePrizes2014
+  def self.normalize(data)
+    result = {}
+    result[:email_s] = required email_address data[:email_s]
+    result[:name_s] = required stripped data[:name_s]
+    result[:school_name_s] = required stripped data[:school_name_s]
+    result[:school_address_s] = required stripped data[:school_address_s]
+    result[:school_type_s] = required enum(data[:school_type_s].to_s.strip.downcase, ['elementary', 'middle_school', 'high_school'])
+    result[:qualifying_school_b] = required stripped data[:qualifying_school_b]
+    result[:title_I_school_b] = nil_if_empty stripped data[:title_I_school_b]
```

Comment:
so normalize is just a ruby method, nothing magical. Since you're overriding it in this class it won't call the superclass's method unless you want it to, so it knows nothing about what HocHardwarePrizes2014#normalize is doing. So you do need to have all of the fields in the form taken care of here.

if you wanted to avoid duplicating the code from HocHardwarePrizes2014#normalize, it looks like you could do something like:
````
result = superclass.normalize(data)
result[:your_new_field1] = required whatever data[:your_new_field1]
result[:your_new_field2] = required whatever data[:your_new_field2]
````